hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -355,18 +355,7 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 			// User-defined AMIs cannot be validated
 			removeStatusCondition(&nodePool.Status.Conditions, hyperv1.NodePoolValidAMIConditionType)
 		} else {
-			// TODO: Should the region be included in the NodePool platform information?
-			ami, err = defaultNodePoolAMI(hcluster.Spec.Platform.AWS.Region, releaseImage)
-			if err != nil {
-				setStatusCondition(&nodePool.Status.Conditions, hyperv1.NodePoolCondition{
-					Type:               hyperv1.NodePoolValidAMIConditionType,
-					Status:             corev1.ConditionFalse,
-					Reason:             hyperv1.NodePoolValidationFailedConditionReason,
-					Message:            fmt.Sprintf("Couldn't discover an AMI for release image %q: %s", nodePool.Spec.Release.Image, err.Error()),
-					ObservedGeneration: nodePool.Generation,
-				})
-				return ctrl.Result{}, fmt.Errorf("couldn't discover an AMI for release image: %w", err)
-			}
+			ami = hcluster.Spec.Platform.AWS.DefaultNodepoolAMI
 			setStatusCondition(&nodePool.Status.Conditions, hyperv1.NodePoolCondition{
```

Comment:
nodepool AMI and hosted cluster AMI should be two different things? they need not depend on each other and they would depend on the arch of the nodepools/workers